internal/database/mysql_test.go

```diff
@@ -237,16 +250,11 @@ func TestDeleteUser(t *testing.T) {
 	subject, probe := setup(t)
 	defer teardown(t, subject, probe)
 
-	err := subject.AddUser("delete_user", "delete_user@example.com", "delete_pass")
-	if err != nil {
-		t.Fatal("Failed to add user -", err)
-	}
-
+	id, err := subject.AddUser("delete", "", "user", "delete_user@example.com",
+		"delete_pass", "2000-01-01")
 	// Verify the user was added
-	var id int
-	err = probe.QueryRow("SELECT id FROM users WHERE username = ?", "delete_user").Scan(&id)
 	if err != nil || id == 0 {
```

Comment:
Since the "AddUser" unit tests should ensure that AddUser works perfectly, we only need yto use "err != nil", no need to check the id.
Also, try to combine with the line above to avoid leaking the error to this scope.

Reply:
Can't do that because that case id is only saved in if loop.

Reply:
removed id==0 tho!